android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetActivity.java

```diff
@@ -37,6 +38,28 @@
 
     protected static final String TAG = JitsiMeetActivity.class.getSimpleName();
 
+    public static final String ACTION_JITSI_MEET_CONFERENCE = "org.jitsi.meet.CONFERENCE";
+    public static final String JITSI_MEET_CONFERENCE_OPTIONS = "JitsiMeetConferenceOptions";
+
+    // Helpers for starting the activity
+    //
+
+    public static void launch(Context context, JitsiMeetConferenceOptions options) {
```

Comment:
are those methods used anywhere currently ?

Reply:
No. They are intended for SDK users. I got the suggestion from Marius (8x8) to simplify the intent creation and stuff.

Here is how simple it becomes for SDK consumers (sample app I made):

```java
    public void onButoonClick(View v) {
        EditText editText = findViewById(R.id.conferenceName);
        String text = editText.getText().toString();

        if (text.length() > 0) {
            JitsiMeetActivity.launch(this, text);
        }
    }
```